index.rst

```diff
@@ -5,18 +5,51 @@ Welcome to awspub's documentation!
 It handles image publication for a given source file (usually a .vmdk file).
 Only EBS backed images/AMIs are supported!
 
+
+----
+
+In this documentation
```

Comment:
This should ideally be just two sections: How-to guides (containing installation and howto_publish) and Reference (containing architecture, reference_index and reference_config). For this you'll need to change the directory structure, and include index files for the two sub-folders corresponding to How-to guides and Reference. The toctree will also have to be corrected appropriately.

Otherwise, at least change the organisation to:

.. grid:: 1 1 2 2
   :padding: 0

   .. grid-item:: :doc:`Installation <docs/installation>`

       **Installation** covers how to setup and install awspub     
       
   .. grid-item:: :doc:`How to publish <docs/howto_publish>`

       **Publish** covers how to publish images to AWS

.. grid:: 1 1 2 2
   :padding: 0

   .. grid-item:: :doc:`Architecture <docs/architecture>`

       **Architecture** covers architectural decisions and considerations

   .. grid-item:: :doc:`Reference <docs/reference_index>`

       **Reference** Auto-generated reference pages for all classes and methods

       **ConfigModel Reference** Auto-generated reference pages for backing config models